katsdptelstate/redis.py

```diff
@@ -84,6 +88,14 @@ def __init__(self, client):
             # redis.TimeoutError: bad host
             # redis.ConnectionError: good host, bad port
             raise ConnectionError("could not connect to redis server: {}".format(e))
+        self._get_script = self._register_script('get.lua')
```

Comment:
I was wondering if this shouldn't be replaced with a pkg_resources.resource_listdir and then register each found script adding it's name as a member. You could then have an execute_script wrapper for each call that handles things missing or not registering.

Reply:
I've changed things to use a loop and store scripts in a dictionary. I prefer to explicitly list the scripts rather than rely on resource_listdir so that if the installation is broken one finds out immediately rather than only when trying to use one of the scripts.